frontend/tests/features/wheresMyGene/landingPageFilter.test.ts

```diff
@@ -35,59 +39,70 @@ describe("Left side bar", () => {
   });
 
   [
-    ["dataset-filter"],
-    ["disease-filter"],
-    ["self-reported-ethnicity-filter"],
-    ["sex-filter"],
-  ].forEach(([filterOption]) => {
+    "dataset-filter",
+    "disease-filter",
+    "self-reported-ethnicity-filter",
+    "sex-filter",
+  ].forEach((filterOption) => {
     test(`Should be able select and de-select options for ${filterOption} filter`, async ({
       page,
     }) => {
-      await goToWMGWithSeededState(page);
+      await goToPage(WMG_WITH_SEEDED_GENES.URL, page);
+
+      // (thuang): This expands the first tissue
+      await page.getByTestId(CELL_TYPE_LABELS_ID).first().click();
 
       await tryUntil(
         async () => {
           // check the count of source data displayed before adding a filter
           const countBeforeFilter = await checkSourceData(page);
 
-          //check plot height before adding a filter
-          const plotSizeBeforeFilter = await checkPlotSize(page);
+          // verify source data loading some data
+          expect(countBeforeFilter).toBeGreaterThan(0);
 
-          //select a filter
-          await selectSecondaryFilterOption(page, filterOption);
+          // check plot height before adding a filter
+          const plotSizeBeforeFilter = await checkPlotSize(page);
 
-          // check the count of source data displayed after adding a filter
-          const countAfterFilter = await checkSourceData(page);
+          // verify data plot data loading some data
+          expect(plotSizeBeforeFilter).toBeGreaterThan(0);
 
-          //check plot height after adding a filter
-          const plotSizeAfterFilter = await checkPlotSize(page);
+          // select a filter
+          await selectSecondaryFilterOption(page, filterOption);
 
-          // verify source data loading some data
-          expect(countBeforeFilter).toBeGreaterThan(0);
+          await tryUntil(
```

Comment:
since heatmap rendering is CPU dependent, wrap these assertions in a retry block, so the browser has time to re-render the heatmap to the correct state